pysm3/models/websky.py

```diff
@@ -314,3 +314,56 @@ def __init__(
             max_nside=max_nside,
             map_dist=map_dist,
         )
+
+
+class AgoraSZ(Model):
+    def __init__(
+        self,
+        nside,
+        sz_type="kinetic",
+        max_nside=None,
+        map_dist=None,
+    ):
+
+        if max_nside is None:
+            if sz_type == "kinetic":
+                max_nside = 8192
+            if sz_type == "thermal":
+                max_nside = 8192
+        super().__init__(nside=nside, max_nside=max_nside, map_dist=map_dist)
+        self.sz_type = sz_type
+        # self.remote_data = utils.RemoteData()
+        # filename = self.remote_data.get(self.get_filename())
+        filename = self.get_filename()
+        self.m = self.read_map(filename, field=0, unit=u.uK_CMB)
+
+    def get_filename(self):
+        """Get SZ filenames for an agora version"""
+
+        if self.sz_type == "kinetic":
+            path = "/Users/kristen/Documents/GitHub/ACT-Simulations/agora/agora_lkszNGbahamas80_nside8192_uk.fits"
+        elif self.sz_type == "thermal":
+            path = "/Users/kristen/Documents/GitHub/ACT-Simulations/agora/agora_ltszNGbahamas80_nside8192_uk.fits"
```

Comment:
@kmsurrao are those maps already available publicly? if not can you copy them to NERSC in a folder accessible by the `cmb` group?

Reply:
Yes, these maps are publicly available [here](https://yomori.github.io/agora/index.html) under tSZ and kSZ in "Simulation Products"

Reply:
@kmsurrao I tried logging in both with my globus ID and with my NERSC account but get access denied.
Can you copy the maps to NERSC in a folder accessible to the `cmb` group?

Reply:
@kmsurrao also, are the units `uK_CMB`?

Reply:
Yes, the maps are in `uK_CMB`. Hmm this is strange, I now get access denied too. I'll look into this and get back to you

Reply:
yuuki should confirm but i think the globus link is pointing either to UChicago / Argonne computers.